platforms/quorum/configuration/roles/setup/geth-bootnode/Readme.md

```diff
@@ -0,0 +1,71 @@
+[//]: # (##############################################################################################)
+[//]: # (Copyright Accenture. All Rights Reserved.)
+[//]: # (SPDX-License-Identifier: Apache-2.0)
+[//]: # (##############################################################################################)
+
+## ROLE: setup/geth-bootnode
+This role installs the geth and bootnode binary if it doesn't exist
+
+### Tasks
+
+#### 1. Create temporary directory
+This tasks creates a temporary directory 
+
+**tempfile**: This module creates a temporary directory.
+
+##### Output Variables
+
+    tmp_directory: This variable stores the info about the temporary directory it created.
+
+#### 2. Check geth
+This task check if geth is already installed or not
+
+**stat**: This module checks if geth is installed or not with path variable.
+
+##### Output Variables
+    geth_stat_result: This variable checks for the presence of file/directory.
+
+#### 3. Check bootnode
+This task check if bootnode is already installed or not
+
+**stat**: This module checks if bootnode is installed or not with path variable.
+
+##### Output Variables
+    bootnode_stat_result: This variable stores the info on availibility on bootnode binary
+
+#### 4. Download geth tar
+This task downloads the geth tar
+
+##### Input Variables
+    repo: The path where the geth binary should be installed
+    dest: Path where the repository should be placed
+
+**when**: It runs when *geth_stat_result.stat.exists* == False i.e. when either geth is not installed ,network.config.consensus == 'ibft' and  *bootnode_stat_result.stat.exists* == False, i.e. when bootnode is not installed.
+
+#### 5. Create bin directory
+This task creates the bin directory if it does not exist.
+
+**file**: This module creates the bin directory if it does not exist.
+
+**when**: It runs when *geth_stat_result.stat.exists* == False, i.e. when geth is not installed and *bootnode_stat_result.stat.exists* == False, i.e. when bootnode is not installed.
```

Comment:
Incorrect, It runs when *geth_stat_result.stat.exists* == False, i.e. when geth is not installed """""""or""""""" 
 *bootnode_stat_result.stat.exists* == False, i.e. when bootnode is not installed.